content/site-map.yaml

```diff
@@ -154,6 +154,8 @@
               file: "/events/ashg-workshop-genomic-analysis-with-galaxy"
             - name: ""
               file: "/events/aises-workshop-genomic-analysis-with-galaxy-workbench"
+            - name: ""
```

Comment:
Hi @jpaten would you mind relocating the new office hours in the site-map to the top of the list - above the 2022 November office hours so that the prev/next page links are ordered chronologically?